edsdme/blocks/feedback/feedback.js

```diff
@@ -192,11 +192,24 @@ async function renderDialog(feedbackButton, formDefinitionUrl, config) {
     const comment = textarea.value;
     const page = window.location.pathname;
     const timestamp = new Date().toISOString();
+    let userName = '';
+    let userEmail = '';
+    if (partnerIsSignedIn()) {
+      try {
+        const profileData = getPartnerDataCookieObject(getCurrentProgramType());
+        userName = `${profileData.firstName} ${profileData.lastName}`;
+        userEmail = profileData.email;
+      } catch (error) {
+        throw new Error(`Failed to parse CPP cookie:: ${error}`);
```

Comment:
I think we should simply log this, throwing an Error here could break the whole page

Reply:
@cod17828 we could set userName and userEmail to a fixed value (e.g. 'invalid') so we even see if/how often this happens. It should never happen.